algorithms/src/Kokkos_StdAlgorithms.hpp

```diff
@@ -48,55 +48,91 @@
 /// \file Kokkos_StdAlgorithms.hpp
 /// \brief Kokkos counterparts for Standard C++ Library algorithms
 
-#include <std_algorithms/Kokkos_Constraints.hpp>
-#include <std_algorithms/Kokkos_RandomAccessIterator.hpp>
+#include <std_algorithms/impl/Kokkos_Constraints.hpp>
+#include <std_algorithms/impl/Kokkos_RandomAccessIterator.hpp>
 #include <std_algorithms/Kokkos_BeginEnd.hpp>
 
 // distance
 #include <std_algorithms/Kokkos_Distance.hpp>
 
-// swap, iter_swap
-#include "std_algorithms/Kokkos_ModifyingOperations.hpp"
+// note that we categorize below the headers
+// following the std classification.
 
-// find, find_if, find_if_not
-// for_each, for_each_n
-// mismatch
-// equal
-// count_if, count
-// all_of, any_of, none_of
-// adjacent_find
-// lexicographical_compare
-// search, search_n
-// find_first_of, find_end
-#include <std_algorithms/Kokkos_NonModifyingSequenceOperations.hpp>
+// modifying ops
+#include "std_algorithms/Kokkos_swap.hpp"
+#include "std_algorithms/Kokkos_iter_swap.hpp"
```

Comment:
Why do you mix double quotes and angle brackets includes?

Reply:
Should I just use angle everywhere?

Reply:
Yes I saw double quotes in other places but I agree with you. I will just use angle.

Reply:
not fixed yet ...

Reply:
btw, as I am fixing this, I am seeing some headaches from having to use <> in some places because I have to specify the correct path for the files from the top-level include. 
So for example for this structure:
```
algorithms/
   src/
       std_algorithms/
            impl/
            Kokkos_AdjacentDifference.hpp 
            ...
```
the include needed for `Kokkos_AdjacentDifference.hpp` is: 
```
#include <std_algorithms/impl/Kokkos_AdjacentDifference.hpp>
#include <std_algorithms/Kokkos_BeginEnd.hpp>
```

which is a bit long, since I could use: 
```
#include "./impl/Kokkos_AdjacentDifference.hpp"
#include "../Kokkos_BeginEnd.hpp"
```
which will always work and not having to worry about what include paths are specified when compiling.

Reply:
double quote is fine for relative path